polymath/qube.py

```
@@ -785,10 +785,10 @@ def _suitable_dtype(cls, dtype='float', opstr=''):
                 return cls._suitable_dtype('float', opstr=opstr)
             if kind in ('i', 'u'):
                 return cls._suitable_dtype('int', opstr=opstr)
-            if kind == 'b':
+            if kind == 'b':  # pragma: no cover
                 return cls._suitable_dtype('bool', opstr=opstr)
 
-        _in_opstr = ' in ' + opstr if opstr else ''                             # noqa
+        _in_opstr = ' in ' + opstr if opstr else ''  # noqa
```

Ruff (0.14.8)

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)